server/request_processor.go

```diff
@@ -33,18 +36,25 @@ type RpcRequest struct {
 	origin                     string
 	isWhitehatBundleCollection bool
 	whitehatBundleId           string
+	id                         uuid.UUID
```

Comment:
not needed here, should be already set on the record entry

Reply:
updated